catala/aixecar/guerrer/guerrer.md

```diff
@@ -0,0 +1,7 @@
+Et sents acorralat i penses que l'única forma de sortir d'aquí és transformarte en super guerrer...
+
+Respires profundament i comences a cridar com tants cops has vist per televisió...
+
+De sobte, un fort soroll et retorna a la realitat. Et despertes al llit de casa teva i te'n adones que t'has cagat a sobre!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.